libs/newsletter/src/driver/hubspot/models/config.ts

```diff
@@ -0,0 +1,10 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
```

Comment:
Why is this injectable?

Reply:
To my understanding, it is Injectable for when it is called using `useValue` for the `DaffNewsletterConfig`